### DISCUSSION TITLE
[mini.ai] How to go to the last argument in a function?

### DISCUSSION THREAD

**Discussion by AlgusDark** [nvim-mini/mini.nvim#404] [Q&A] [answered]
I see myself trying to go to the last argument very often. I know the `goto_right`, and I can use a (Argument), but I wonder if there is an easier way:
- Go to last argument
  - Paste

Or
- Change around last argument
  - Insert mode after the last argument [from  `| fn (a,b,c)` to `fn (a,b,c|)`]

**Comment by echasnovski** [comment, marked as answer]
Not really possible with current design and implementation of `argument` textobject in 'mini.ai'.

I can suggest using `sf` (`find`) functionality of 'mini.surround' to go to the right part of function call with `sff`.

**Reply by AlgusDark** [reply]
Hey, that's a good suggestion. I already use 'mini.surround'. Thank you for the awesome packages and the solution :D
